python_modules/libraries/dagster-github/dagster_github/resources.py

```diff
@@ -185,7 +180,7 @@ class GithubResource(ConfigurableResourceFactory[GithubClient]):
         ),
     )
 
-    def create_resource(self, context: InitResourceContextWithKeyMapping) -> GithubClient:
+    def get_github_client(self) -> GithubClient:
```

Comment:
`get_client`? what pattern do we want to follow here?

Reply:
IMO when there is a ClassName we should have a method `get_class_name()`

Reply:
However I wasn't consistent there as it should have been `get_datadog_client` in the datadog diff